<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Standardise variants output | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq</title>
  <meta name="description" content="5 Standardise variants output | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  <meta name="generator" content="bookdown 0.16 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Standardise variants output | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Standardise variants output | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  
  
  

<meta name="author" content="Anna Quaglieri" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="4-ch-acs.html"/>
<link rel="next" href="6-ch-tcga-down.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Supporting scripts for: Finding a suitable library size to call variants with RNA-Seq</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Setup</a></li>
<li class="chapter" data-level="2" data-path="2-software-used.html"><a href="2-software-used.html"><i class="fa fa-check"></i><b>2</b> Software used</a></li>
<li class="chapter" data-level="3" data-path="3-ch-leucegene-down.html"><a href="3-ch-leucegene-down.html"><i class="fa fa-check"></i><b>3</b> Download, quality control and downsample the Leucegene RNA-Seq samples</a><ul>
<li class="chapter" data-level="3.1" data-path="3-ch-leucegene-down.html"><a href="3-ch-leucegene-down.html#download-rna-seq-data-from-geo"><i class="fa fa-check"></i><b>3.1</b> Download RNA-Seq data from GEO</a><ul>
<li class="chapter" data-level="3.1.1" data-path="3-ch-leucegene-down.html"><a href="3-ch-leucegene-down.html#get-srx-sample-names"><i class="fa fa-check"></i><b>3.1.1</b> Get SRX sample names</a></li>
<li class="chapter" data-level="3.1.2" data-path="3-ch-leucegene-down.html"><a href="3-ch-leucegene-down.html#create-ncbi-query"><i class="fa fa-check"></i><b>3.1.2</b> Create NCBI query</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="3-ch-leucegene-down.html"><a href="3-ch-leucegene-down.html#ch:leucegene-downsample"><i class="fa fa-check"></i><b>3.2</b> Downsampling FASTQ files to a fixed number of reads</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-ch-acs.html"><a href="4-ch-acs.html"><i class="fa fa-check"></i><b>4</b> Align, call and standardise</a><ul>
<li class="chapter" data-level="4.1" data-path="4-ch-acs.html"><a href="4-ch-acs.html#ch:align"><i class="fa fa-check"></i><b>4.1</b> Alignement, Read Groups, Mark duplicates</a><ul>
<li class="chapter" data-level="4.1.1" data-path="4-ch-acs.html"><a href="4-ch-acs.html#star-index"><i class="fa fa-check"></i><b>4.1.1</b> STAR index</a></li>
<li class="chapter" data-level="4.1.2" data-path="4-ch-acs.html"><a href="4-ch-acs.html#star-1pass"><i class="fa fa-check"></i><b>4.1.2</b> STAR-1pass</a></li>
<li class="chapter" data-level="4.1.3" data-path="4-ch-acs.html"><a href="4-ch-acs.html#star-2pass"><i class="fa fa-check"></i><b>4.1.3</b> STAR-2pass</a></li>
<li class="chapter" data-level="4.1.4" data-path="4-ch-acs.html"><a href="4-ch-acs.html#sec:post-align"><i class="fa fa-check"></i><b>4.1.4</b> Processing post alingment</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="4-ch-acs.html"><a href="4-ch-acs.html#gatk-pre-processing-to-call-variants-with-mutect-and-varscan"><i class="fa fa-check"></i><b>4.2</b> GATK pre-processing to call variants with MuTect and VarScan</a><ul>
<li class="chapter" data-level="4.2.1" data-path="4-ch-acs.html"><a href="4-ch-acs.html#splitncigarreads"><i class="fa fa-check"></i><b>4.2.1</b> SplitNCigarReads</a></li>
<li class="chapter" data-level="4.2.2" data-path="4-ch-acs.html"><a href="4-ch-acs.html#base-recalibration"><i class="fa fa-check"></i><b>4.2.2</b> Base recalibration</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="4-ch-acs.html"><a href="4-ch-acs.html#ch:call-variants"><i class="fa fa-check"></i><b>4.3</b> Variant calling</a><ul>
<li class="chapter" data-level="4.3.1" data-path="4-ch-acs.html"><a href="4-ch-acs.html#mutect2-settings"><i class="fa fa-check"></i><b>4.3.1</b> MuTect2 settings</a></li>
<li class="chapter" data-level="4.3.2" data-path="4-ch-acs.html"><a href="4-ch-acs.html#samtools-varscan2-settings"><i class="fa fa-check"></i><b>4.3.2</b> Samtools + VarScan2 settings</a></li>
<li class="chapter" data-level="4.3.3" data-path="4-ch-acs.html"><a href="4-ch-acs.html#vardict-settings"><i class="fa fa-check"></i><b>4.3.3</b> VarDict settings</a></li>
<li class="chapter" data-level="4.3.4" data-path="4-ch-acs.html"><a href="4-ch-acs.html#vep-settings"><i class="fa fa-check"></i><b>4.3.4</b> VEP settings</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-standardise-variants-output.html"><a href="5-standardise-variants-output.html"><i class="fa fa-check"></i><b>5</b> Standardise variants output</a></li>
<li class="chapter" data-level="6" data-path="6-ch-tcga-down.html"><a href="6-ch-tcga-down.html"><i class="fa fa-check"></i><b>6</b> TCGA-LAML downsampling</a></li>
<li class="chapter" data-level="7" data-path="7-ch-refer.html"><a href="7-ch-refer.html"><i class="fa fa-check"></i><b>7</b> References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Proudly published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supporting scripts: Finding a suitable library size to call variants in RNA-Seq</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="standardise-variants-output" class="section level1">
<h1><span class="header-section-number">5</span> Standardise variants output</h1>
<p>The function <code>Rscript ./functions/call_variants.R</code> will return as output variants in a standardised format so that they can be easily compared across callers. This means that fields of interest, like variant allele frequency and depth at a variant site, which are usually provided in with different names across callers, are returned with the same names. The standardised variants are returned as <code>.txt</code> files. The functions in <code>call_variants.R</code> aimed at standardising variants are also the same ones present in the <code>varikondo</code><span class="citation">(Quaglieri and Flensburg, <a href="7-ch-refer.html#ref-Quaglieri_undated-yh">n.d.</a>)</span> R package available on github. Examples on how to use its functions are provided in the package website <a href="https://annaquaglieri16.github.io/varikondo/articles/vignette.html" class="uri">https://annaquaglieri16.github.io/varikondo/articles/vignette.html</a> and below.</p>
<p>The earlier version of the functions returns some duplicated rows compared to using <code>varikondo</code> but this won’t affect the results of the study as we only looked at unique variants.</p>
<p>The <code>.txt</code> files <code>SRX729580_germline_final.txt</code> produced for each caller are used for sensitivity analysis and they are processed as discussed in the Methods paper.</p>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb31-1" data-line-number="1"><span class="co"># devtools::install_github(&quot;annaquaglieri16/varikondo&quot;)</span></a>
<a class="sourceLine" id="cb31-2" data-line-number="2"></a>
<a class="sourceLine" id="cb31-3" data-line-number="3"><span class="co"># detach(&quot;package:varikondo&quot;)</span></a>
<a class="sourceLine" id="cb31-4" data-line-number="4"><span class="kw">library</span>(varikondo)</a>
<a class="sourceLine" id="cb31-5" data-line-number="5">utils<span class="op">::</span><span class="kw">packageVersion</span>(<span class="st">&quot;varikondo&quot;</span>)</a></code></pre></div>
<pre><code>## [1] &#39;0.10.8&#39;</code></pre>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb33-1" data-line-number="1"><span class="co"># Variant file created with ./functions/call_variants.R</span></a>
<a class="sourceLine" id="cb33-2" data-line-number="2">mutect.table &lt;-<span class="st"> </span><span class="kw">read.delim</span>(<span class="st">&quot;data/test-variants/mutect/regions_caller_defaults/annotated_variants/SRX729580_germline_final.txt&quot;</span>,<span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>)</a>
<a class="sourceLine" id="cb33-3" data-line-number="3"><span class="kw">dim</span>(mutect.table)</a></code></pre></div>
<pre><code>## [1] 3005   82</code></pre>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb35-1" data-line-number="1"><span class="kw">sum</span>(<span class="kw">duplicated</span>(mutect.table))</a></code></pre></div>
<pre><code>## [1] 2</code></pre>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb37-1" data-line-number="1"><span class="co"># Read and parse with varikondo</span></a>
<a class="sourceLine" id="cb37-2" data-line-number="2">mutect.varik &lt;-<span class="st"> </span>varikondo<span class="op">::</span><span class="kw">parse_vcf_output</span>(<span class="st">&quot;data/test-variants/mutect/regions_caller_defaults/annotated_variants/SRX729580_germline_annotated.vcf&quot;</span>, <span class="dt">caller =</span> <span class="st">&quot;mutect&quot;</span>,<span class="dt">vep=</span><span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb37-3" data-line-number="3"><span class="kw">dim</span>(mutect.varik)</a></code></pre></div>
<pre><code>## [1] 3003   87</code></pre>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb39-1" data-line-number="1"><span class="co"># Variant file created with ./functions/call_variants.R</span></a>
<a class="sourceLine" id="cb39-2" data-line-number="2">varscan.table &lt;-<span class="st"> </span><span class="kw">read.delim</span>(<span class="st">&quot;data/test-variants/varscan/regions_caller_defaults/annotated_variants/SRX729580_germline_final.txt&quot;</span>,<span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>)</a>
<a class="sourceLine" id="cb39-3" data-line-number="3"><span class="kw">dim</span>(varscan.table)</a></code></pre></div>
<pre><code>## [1] 1031   82</code></pre>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb41-1" data-line-number="1"><span class="co"># Read and parse with varikondo</span></a>
<a class="sourceLine" id="cb41-2" data-line-number="2">varscan.varik &lt;-<span class="st"> </span>varikondo<span class="op">::</span><span class="kw">parse_vcf_output</span>(<span class="st">&quot;data/test-variants/varscan/regions_caller_defaults/annotated_variants/SRX729580_germline_annotated.vcf&quot;</span>, <span class="dt">caller =</span> <span class="st">&quot;varscan&quot;</span>,<span class="dt">vep =</span> <span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb41-3" data-line-number="3"><span class="kw">dim</span>(varscan.varik)</a></code></pre></div>
<pre><code>## [1] 1031   87</code></pre>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb43-1" data-line-number="1"><span class="co"># VarDict file created with ./functions/call_variants.R</span></a>
<a class="sourceLine" id="cb43-2" data-line-number="2">vardict.table &lt;-<span class="st"> </span><span class="kw">read.delim</span>(<span class="st">&quot;data/test-variants/vardict/regions_caller_defaults/annotated_variants/SRX729580_germline_final.txt&quot;</span>,<span class="dt">sep =</span> <span class="st">&quot;</span><span class="ch">\t</span><span class="st">&quot;</span>)</a>
<a class="sourceLine" id="cb43-3" data-line-number="3">vardict.table<span class="op">$</span>key &lt;-<span class="st"> </span><span class="kw">paste</span>(vardict.table<span class="op">$</span>Location,vardict.table<span class="op">$</span>ref,</a>
<a class="sourceLine" id="cb43-4" data-line-number="4">                           vardict.table<span class="op">$</span>alt,vardict.table<span class="op">$</span>Existing_variation,vardict.table<span class="op">$</span>Consequence,<span class="dt">sep=</span><span class="st">&quot;.&quot;</span>)</a>
<a class="sourceLine" id="cb43-5" data-line-number="5"></a>
<a class="sourceLine" id="cb43-6" data-line-number="6"><span class="kw">dim</span>(vardict.table)</a></code></pre></div>
<pre><code>## [1] 2799   84</code></pre>
<div class="sourceCode" id="cb45"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb45-1" data-line-number="1"><span class="kw">sum</span>(<span class="kw">duplicated</span>(vardict.table))</a></code></pre></div>
<pre><code>## [1] 4</code></pre>
<div class="sourceCode" id="cb47"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb47-1" data-line-number="1"><span class="co"># Read and parse with varikondo</span></a>
<a class="sourceLine" id="cb47-2" data-line-number="2">vardict.varik &lt;-<span class="st"> </span>varikondo<span class="op">::</span><span class="kw">parse_vcf_output</span>(<span class="st">&quot;data/test-variants/vardict/regions_caller_defaults/annotated_variants/SRX729580_germline_annotated.vcf&quot;</span>, <span class="dt">caller =</span> <span class="st">&quot;vardict&quot;</span>,<span class="dt">vep =</span> <span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb47-3" data-line-number="3"><span class="kw">dim</span>(vardict.varik)</a></code></pre></div>
<pre><code>## [1] 2795   85</code></pre>
<div class="sourceCode" id="cb49"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb49-1" data-line-number="1">vardict.varik<span class="op">$</span>key &lt;-<span class="st"> </span><span class="kw">paste</span>(vardict.varik<span class="op">$</span>Location,vardict.varik<span class="op">$</span>ref,</a>
<a class="sourceLine" id="cb49-2" data-line-number="2">                           vardict.varik<span class="op">$</span>alt,vardict.varik<span class="op">$</span>Existing_variation,vardict.varik<span class="op">$</span>Consequence,<span class="dt">sep=</span><span class="st">&quot;.&quot;</span>)</a></code></pre></div>
<div class="sourceCode" id="cb50"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb50-1" data-line-number="1"><span class="co"># All variants parsed with the previous function are also present when using varikondo</span></a>
<a class="sourceLine" id="cb50-2" data-line-number="2">vardict.table<span class="op">$</span>key[<span class="op">!</span>(vardict.table<span class="op">$</span>key <span class="op">%in%</span><span class="st"> </span>vardict.varik<span class="op">$</span>key)]</a></code></pre></div>
<pre><code>## character(0)</code></pre>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="4-ch-acs.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="6-ch-tcga-down.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/annaquaglieri16/Supporting-scripts-library-size-RNA-Seq/tree/master/Rmd/02-align-call-standardise.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Supporting-libsize-scripts-book.pdf"],
"toc": {
"collapse": "none"
}
});
});
</script>

</body>

</html>
