<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>3 Align, call and standardise | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq</title>
  <meta name="description" content="3 Align, call and standardise | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  <meta name="generator" content="bookdown 0.16 and GitBook 2.6.7" />

  <meta property="og:title" content="3 Align, call and standardise | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="3 Align, call and standardise | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  
  
  

<meta name="author" content="Anna Quaglieri" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="ch-leucegene-down.html"/>
<link rel="next" href="standardise-variants-output.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Setup</a></li>
<li class="chapter" data-level="2" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html"><i class="fa fa-check"></i><b>2</b> Download, quality control and downsample the Leucegene RNA-Seq samples</a><ul>
<li class="chapter" data-level="2.1" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#ch:leucegene-download"><i class="fa fa-check"></i><b>2.1</b> Download RNA-Seq data from GEO</a><ul>
<li class="chapter" data-level="2.1.1" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#get-srx-sample-names"><i class="fa fa-check"></i><b>2.1.1</b> Get SRX sample names</a></li>
<li class="chapter" data-level="2.1.2" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#create-ncbi-query"><i class="fa fa-check"></i><b>2.1.2</b> Create NCBI query</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#ch:leucegene-downsample"><i class="fa fa-check"></i><b>2.2</b> Downsampling FASTQ files to a fixed number of reads</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch-acs.html"><a href="ch-acs.html"><i class="fa fa-check"></i><b>3</b> Align, call and standardise</a><ul>
<li class="chapter" data-level="3.1" data-path="ch-acs.html"><a href="ch-acs.html#ch:align"><i class="fa fa-check"></i><b>3.1</b> Alignement, Read Groups, Mark duplicates</a><ul>
<li class="chapter" data-level="3.1.1" data-path="ch-acs.html"><a href="ch-acs.html#star-index"><i class="fa fa-check"></i><b>3.1.1</b> STAR index</a></li>
<li class="chapter" data-level="3.1.2" data-path="ch-acs.html"><a href="ch-acs.html#star-1pass"><i class="fa fa-check"></i><b>3.1.2</b> STAR-1pass</a></li>
<li class="chapter" data-level="3.1.3" data-path="ch-acs.html"><a href="ch-acs.html#star-2pass"><i class="fa fa-check"></i><b>3.1.3</b> STAR-2pass</a></li>
<li class="chapter" data-level="3.1.4" data-path="ch-acs.html"><a href="ch-acs.html#sec:post-align"><i class="fa fa-check"></i><b>3.1.4</b> Processing post alingment</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="ch-acs.html"><a href="ch-acs.html#gatk-pre-processing-to-call-variants-with-mutect-and-varscan"><i class="fa fa-check"></i><b>3.2</b> GATK pre-processing to call variants with MuTect and VarScan</a><ul>
<li class="chapter" data-level="3.2.1" data-path="ch-acs.html"><a href="ch-acs.html#splitncigarreads"><i class="fa fa-check"></i><b>3.2.1</b> SplitNCigarReads</a></li>
<li class="chapter" data-level="3.2.2" data-path="ch-acs.html"><a href="ch-acs.html#base-recalibration"><i class="fa fa-check"></i><b>3.2.2</b> Base recalibration</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="ch-acs.html"><a href="ch-acs.html#ch:call-variants"><i class="fa fa-check"></i><b>3.3</b> Variant calling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="ch-acs.html"><a href="ch-acs.html#mutect2-settings"><i class="fa fa-check"></i><b>3.3.1</b> MuTect2 settings</a></li>
<li class="chapter" data-level="3.3.2" data-path="ch-acs.html"><a href="ch-acs.html#samtools-varscan2-settings"><i class="fa fa-check"></i><b>3.3.2</b> Samtools + VarScan2 settings</a></li>
<li class="chapter" data-level="3.3.3" data-path="ch-acs.html"><a href="ch-acs.html#vardict-settings"><i class="fa fa-check"></i><b>3.3.3</b> VarDict settings</a></li>
<li class="chapter" data-level="3.3.4" data-path="ch-acs.html"><a href="ch-acs.html#vep-settings"><i class="fa fa-check"></i><b>3.3.4</b> VEP settings</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="standardise-variants-output.html"><a href="standardise-variants-output.html"><i class="fa fa-check"></i><b>4</b> Standardise variants output</a></li>
<li class="chapter" data-level="5" data-path="ch-tcga-down.html"><a href="ch-tcga-down.html"><i class="fa fa-check"></i><b>5</b> TCGA-LAML downsampling</a></li>
<li class="chapter" data-level="6" data-path="ch-refer.html"><a href="ch-refer.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supporting scripts: Finding a suitable library size to call variants in RNA-Seq</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch:acs" class="section level1">
<h1><span class="header-section-number">3</span> Align, call and standardise</h1>
<div id="ch:align" class="section level2">
<h2><span class="header-section-number">3.1</span> Alignement, Read Groups, Mark duplicates</h2>
<p>The <em>FASTQ</em> files were aligned with <em>STAR</em>. The index needed for STAR is created using the functions below after downloading the UCSC hg19 fasta reference genome and GTF file from <a href="https://sapac.support.illumina.com/sequencing/sequencing_software/igenome.html">iGenomes</a>. For the index, STAR requires an extra parameter called <code>sjdbOverhang</code> which is usually set to be <em>(read length - 1)</em> epends on the read length used for sequencing, in this case 100bp - 1.</p>
<div id="star-index" class="section level3">
<h3><span class="header-section-number">3.1.1</span> STAR index</h3>
<div class="sourceCode" id="cb13"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb13-1" data-line-number="1"><span class="ex">module</span> load STAR/2.5</a>
<a class="sourceLine" id="cb13-2" data-line-number="2"><span class="ex">module</span> load R/3.5.2</a>
<a class="sourceLine" id="cb13-3" data-line-number="3"></a>
<a class="sourceLine" id="cb13-4" data-line-number="4"><span class="co"># Iniaitlise Genome directory where to save STAR Index and STAR Fusion Index folders</span></a>
<a class="sourceLine" id="cb13-5" data-line-number="5"><span class="co"># 99 = read length - 1 as suggested in STAR manual </span></a>
<a class="sourceLine" id="cb13-6" data-line-number="6"><span class="va">star_genome100=</span>path_to_genome_directory/star_index_hg19_99</a>
<a class="sourceLine" id="cb13-7" data-line-number="7"><span class="va">genome=</span>reference_genome_hg19.fasta</a>
<a class="sourceLine" id="cb13-8" data-line-number="8"><span class="va">gtf=</span>gtf_hg19.gtf</a>
<a class="sourceLine" id="cb13-9" data-line-number="9"></a>
<a class="sourceLine" id="cb13-10" data-line-number="10"><span class="fu">mkdir</span> -p <span class="va">${star_genome100}</span></a>
<a class="sourceLine" id="cb13-11" data-line-number="11"></a>
<a class="sourceLine" id="cb13-12" data-line-number="12"><span class="ex">STAR</span> \</a>
<a class="sourceLine" id="cb13-13" data-line-number="13">--runMode genomeGenerate \</a>
<a class="sourceLine" id="cb13-14" data-line-number="14">--genomeDir <span class="va">${star_genome100}</span> \</a>
<a class="sourceLine" id="cb13-15" data-line-number="15">--genomeFastaFiles <span class="va">${genome}</span> \</a>
<a class="sourceLine" id="cb13-16" data-line-number="16">--sjdbOverhang 99 \</a>
<a class="sourceLine" id="cb13-17" data-line-number="17">--sjdbGTFfile <span class="va">${gtf}</span></a></code></pre></div>
<p>From this point onwards the script <code>./functions/align-call-stand.sh</code> can be setup and run.</p>
</div>
<div id="star-1pass" class="section level3">
<h3><span class="header-section-number">3.1.2</span> STAR-1pass</h3>
<p>The 45 FASTQ files were aligned using STAR in two pass mode. First all samples are aligned in the first pass using the example function below, a wrapper function that calls STAR. The example is provided for one sample.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb14-1" data-line-number="1"><span class="ex">module</span> load STAR/2.5</a>
<a class="sourceLine" id="cb14-2" data-line-number="2"><span class="ex">module</span> load R/3.5.2</a>
<a class="sourceLine" id="cb14-3" data-line-number="3"></a>
<a class="sourceLine" id="cb14-4" data-line-number="4"><span class="bu">cd</span> ../../</a>
<a class="sourceLine" id="cb14-5" data-line-number="5"></a>
<a class="sourceLine" id="cb14-6" data-line-number="6"><span class="va">star_genome100=</span>path_to_genome_directory/star_index_hg19_99</a>
<a class="sourceLine" id="cb14-7" data-line-number="7"><span class="co"># compress fastq file</span></a>
<a class="sourceLine" id="cb14-8" data-line-number="8"><span class="ex">pigz</span> data/test-fastq/sub*combined*.fastq</a>
<a class="sourceLine" id="cb14-9" data-line-number="9"><span class="va">FQ1=</span>data/test-fastq/sub30M_SRX729580_combined_1.fastq.gz</a>
<a class="sourceLine" id="cb14-10" data-line-number="10"><span class="va">FQ2=</span>data/test-fastq/sub30M_SRX729580_combined_2.fastq.gz</a>
<a class="sourceLine" id="cb14-11" data-line-number="11"><span class="va">bamout=</span>SRX729580</a>
<a class="sourceLine" id="cb14-12" data-line-number="12"><span class="fu">mkdir</span> data/test-fastq/star-pass1</a>
<a class="sourceLine" id="cb14-13" data-line-number="13"></a>
<a class="sourceLine" id="cb14-14" data-line-number="14"><span class="ex">Rscript</span> ./functions/run_STAR.R \</a>
<a class="sourceLine" id="cb14-15" data-line-number="15">--genome_index <span class="va">${star_genome100}</span> \</a>
<a class="sourceLine" id="cb14-16" data-line-number="16">--fastqfiles <span class="va">$FQ1</span>,<span class="va">$FQ2</span> \</a>
<a class="sourceLine" id="cb14-17" data-line-number="17">--sampleName SRX729580 \</a>
<a class="sourceLine" id="cb14-18" data-line-number="18">--RlibPath <span class="st">&quot;path_to_R/3.4&quot;</span> \</a>
<a class="sourceLine" id="cb14-19" data-line-number="19">--outdir data/test-fastq/star-pass1 --STARmode <span class="st">&quot;1Pass&quot;</span></a>
<a class="sourceLine" id="cb14-20" data-line-number="20"></a>
<a class="sourceLine" id="cb14-21" data-line-number="21"><span class="co"># Check alignment stats</span></a>
<a class="sourceLine" id="cb14-22" data-line-number="22"><span class="ex">multiqc</span> data/test-fastq/star-pass1 --interactive -n <span class="st">&quot;star1pass&quot;</span> -o data/test-fastq/star-pass1</a></code></pre></div>
<p>The <code>R</code> function above is a wrapper for the <code>STAR</code> call with settings below:</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb15-1" data-line-number="1"><span class="co"># Version STAR/2.5</span></a>
<a class="sourceLine" id="cb15-2" data-line-number="2"><span class="ex">STAR</span> --genomeDir <span class="va">$star_genome100</span> \</a>
<a class="sourceLine" id="cb15-3" data-line-number="3">--readFilesIn <span class="va">$FQ1</span> <span class="va">$FQ2</span> \</a>
<a class="sourceLine" id="cb15-4" data-line-number="4">--runThreadN 27 \</a>
<a class="sourceLine" id="cb15-5" data-line-number="5">--chimSegmentMin 10 \</a>
<a class="sourceLine" id="cb15-6" data-line-number="6">--readFilesCommand zcat \</a>
<a class="sourceLine" id="cb15-7" data-line-number="7">--alignSJoverhangMin 8 \</a>
<a class="sourceLine" id="cb15-8" data-line-number="8">--outBAMcompression 10 \</a>
<a class="sourceLine" id="cb15-9" data-line-number="9">--alignSJDBoverhangMin 1 \</a>
<a class="sourceLine" id="cb15-10" data-line-number="10">--limitBAMsortRAM 85741557872 \</a>
<a class="sourceLine" id="cb15-11" data-line-number="11">--outFilterMismatchNmax 999 \</a>
<a class="sourceLine" id="cb15-12" data-line-number="12">--alignIntronMin 20 \</a>
<a class="sourceLine" id="cb15-13" data-line-number="13">--alignIntronMax 200000 \</a>
<a class="sourceLine" id="cb15-14" data-line-number="14">--alignMatesGapMax 20000 \</a>
<a class="sourceLine" id="cb15-15" data-line-number="15">--outFileNamePrefix data/test-fastq/star-pass1/SRX729580 \</a>
<a class="sourceLine" id="cb15-16" data-line-number="16">--outSAMtype BAM SortedByCoordinate \</a>
<a class="sourceLine" id="cb15-17" data-line-number="17">--outFilterType BySJout \</a>
<a class="sourceLine" id="cb15-18" data-line-number="18">--outFilterMultimapNmax 15</a></code></pre></div>
<p>After running STAR on all the fastq files available the splice junctions are collected from the first pass and use them for the second pass.</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb16-1" data-line-number="1"><span class="co"># concatenate splice junctions from all samples from ran in pass1</span></a>
<a class="sourceLine" id="cb16-2" data-line-number="2"><span class="fu">cat</span> data/test-fastq/star-pass1/*SJ.out.tab <span class="op">&gt;</span> data/test-fastq/star-pass1/combined_sj.out.tab</a>
<a class="sourceLine" id="cb16-3" data-line-number="3"><span class="co"># Dobin suggests to remove chrm cause they are usually False positives</span></a>
<a class="sourceLine" id="cb16-4" data-line-number="4"><span class="fu">awk</span> <span class="st">&#39;!/chrM/&#39;</span> data/test-fastq/star-pass1/combined_sj.out.tab <span class="op">&gt;</span> data/test-fastq/star-pass1/combined_sj_nochrM.out.tab</a></code></pre></div>
</div>
<div id="star-2pass" class="section level3">
<h3><span class="header-section-number">3.1.3</span> STAR-2pass</h3>
<p>The STAR call aligns and sorts reads by coordinate.</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb17-1" data-line-number="1"><span class="ex">module</span> load STAR/2.5</a>
<a class="sourceLine" id="cb17-2" data-line-number="2"><span class="ex">module</span> load R/3.5.2</a>
<a class="sourceLine" id="cb17-3" data-line-number="3"></a>
<a class="sourceLine" id="cb17-4" data-line-number="4"><span class="va">star_genome100=</span>path_to_genome_directory/star_index_hg19_99</a>
<a class="sourceLine" id="cb17-5" data-line-number="5"><span class="va">FQ1=</span>data/test-fastq/sub30M_SRX729580_combined_1.fastq.gz</a>
<a class="sourceLine" id="cb17-6" data-line-number="6"><span class="va">FQ2=</span>data/test-fastq/sub30M_SRX729580_combined_2.fastq.gz</a>
<a class="sourceLine" id="cb17-7" data-line-number="7"><span class="fu">mkdir</span> data/test-fastq/star-pass2</a>
<a class="sourceLine" id="cb17-8" data-line-number="8"></a>
<a class="sourceLine" id="cb17-9" data-line-number="9"><span class="ex">Rscript</span> ./functions/run_STAR.R \</a>
<a class="sourceLine" id="cb17-10" data-line-number="10">--genome_index <span class="va">$star_genome100</span> \</a>
<a class="sourceLine" id="cb17-11" data-line-number="11">--fastqfiles <span class="va">$FQ1</span>,<span class="va">$FQ2</span> \</a>
<a class="sourceLine" id="cb17-12" data-line-number="12">--sampleName SRX729580 \</a>
<a class="sourceLine" id="cb17-13" data-line-number="13">--RlibPath <span class="st">&quot;path_to_R/3.4&quot;</span> \</a>
<a class="sourceLine" id="cb17-14" data-line-number="14">--outdir data/test-fastq/star-pass2 --STARmode <span class="st">&quot;2PassMulti&quot;</span> \</a>
<a class="sourceLine" id="cb17-15" data-line-number="15">--sjfile data/test-fastq/star-pass1/combined_sj_nochrM.out.tab</a>
<a class="sourceLine" id="cb17-16" data-line-number="16"></a>
<a class="sourceLine" id="cb17-17" data-line-number="17"><span class="co"># Check alignment stats</span></a>
<a class="sourceLine" id="cb17-18" data-line-number="18"><span class="ex">multiqc</span> data/test-fastq/star-pass1 --interactive -n <span class="st">&quot;star2pass&quot;</span> -o data/test-fastq/star-pass2</a></code></pre></div>
<p>The function above is a wrapper for the STAR call used in the first pass with the additinal settings below:</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb18-1" data-line-number="1"><span class="ex">--outFilterScoreMinOverLread</span> 0.3 \</a>
<a class="sourceLine" id="cb18-2" data-line-number="2">--chimSegmentReadGapMax 6 \</a>
<a class="sourceLine" id="cb18-3" data-line-number="3">--alignSJstitchMismatchNmax 5 -1 5 5 \</a>
<a class="sourceLine" id="cb18-4" data-line-number="4">--chimOutType WithinBAM \</a>
<a class="sourceLine" id="cb18-5" data-line-number="5">--chimJunctionOverhangMin 2 \</a>
<a class="sourceLine" id="cb18-6" data-line-number="6">--limitSjdbInsertNsj 2273673 \</a>
<a class="sourceLine" id="cb18-7" data-line-number="7">--sjdbFileChrStartEnd data/test-fastq/star-pass1/combined_sj_nochrM.out.tab</a></code></pre></div>
</div>
<div id="sec:post-align" class="section level3">
<h3><span class="header-section-number">3.1.4</span> Processing post alingment</h3>
<p>These include:</p>
<ol style="list-style-type: decimal">
<li><p>Mark PCR duplicates, <a href="http://lomereiter.github.io/sambamba/docs/sambamba-markdup.html">sambamba markdup</a> from <code>sambamba/0.6.6</code> was used.</p></li>
<li><p>Add Read Groups using <a href="https://broadinstitute.github.io/picard/command-line-overview.html">AddOrReplaceReadGroups</a> (from <code>picard-tools/2.9.4</code>). Read groups are required by <code>GATK</code>.</p></li>
</ol>
<p>If a sample was sequenced across different lanes lane-specific read groups are required for each separate bamfile (e.g. SampleName_L1, SampleName_L2). This sample name will be used for the fields <code>RGLB</code> and <code>RGSM</code> in the <code>AddOrReplaceReadGroups</code> example below.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb19-1" data-line-number="1"><span class="ex">module</span> load picard-tools/2.9.4</a>
<a class="sourceLine" id="cb19-2" data-line-number="2"><span class="co"># Picard tool function to add read groups to a bamfile</span></a>
<a class="sourceLine" id="cb19-3" data-line-number="3"><span class="ex">AddOrReplaceReadGroups</span> \</a>
<a class="sourceLine" id="cb19-4" data-line-number="4">I= data/test-fastq/star-pass2/SRX729580Aligned.sortedByCoord.out.bam \</a>
<a class="sourceLine" id="cb19-5" data-line-number="5">O= data/test-fastq/star-pass2/SRX729580Aligned.sortedByCoord.out.RG.bam \</a>
<a class="sourceLine" id="cb19-6" data-line-number="6">RGID=SRX729580 \</a>
<a class="sourceLine" id="cb19-7" data-line-number="7">RGPU=SRX729580 \</a>
<a class="sourceLine" id="cb19-8" data-line-number="8">RGLB=SRX729580_L1 \</a>
<a class="sourceLine" id="cb19-9" data-line-number="9">RGPL=<span class="st">&quot;illumina&quot;</span> \</a>
<a class="sourceLine" id="cb19-10" data-line-number="10">RGSM=SRX729580_L1</a></code></pre></div>
<ol start="3" style="list-style-type: decimal">
<li><p><a href="https://broadinstitute.github.io/picard/command-line-overview.html">ValidateSamFile</a> (from picard-tools/2.9.4) can be used to check for errors in the final bamfile.</p></li>
<li><p><code>sambamba index</code> to index the final bamfile.</p></li>
</ol>
<div class="sourceCode" id="cb20"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb20-1" data-line-number="1"><span class="ex">module</span> load sambamba/0.6.6</a>
<a class="sourceLine" id="cb20-2" data-line-number="2"></a>
<a class="sourceLine" id="cb20-3" data-line-number="3"><span class="ex">sambamba</span> index data/test-fastq/star-pass2/SRX729580Aligned.sortedByCoord.out.RG.bam</a></code></pre></div>
</div>
</div>
<div id="gatk-pre-processing-to-call-variants-with-mutect-and-varscan" class="section level2">
<h2><span class="header-section-number">3.2</span> GATK pre-processing to call variants with MuTect and VarScan</h2>
<p>The function <code>./functions/gatk_process_pipe.R</code> performs the following steps required by GATK3 to run MuTect2:</p>
<ul>
<li><em>SplitNCigarReads</em> see <a href="https://software.broadinstitute.org/gatk/documentation/tooldocs/3.8-0/org_broadinstitute_gatk_tools_walkers_rnaseq_SplitNCigarReads.php">GATK documentation</a></li>
<li><em>Base recalibration</em> see <a href="https://gatkforums.broadinstitute.org/gatk/discussion/44/base-quality-score-recalibration-bqsr">GATK documentation</a>.</li>
</ul>
<p>These steps are suggested in the <a href="https://software.broadinstitute.org/gatk/documentation/article.php?id=3891"><code>GATK best practices for RNA-Seq variant calling</code></a>.</p>
<p>Below is an example call which wraps the steps above and check if output files have already been created.</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb21-1" data-line-number="1"><span class="ex">module</span> load gatk/3.7.0</a>
<a class="sourceLine" id="cb21-2" data-line-number="2"><span class="ex">module</span> load sambamba/0.6.6</a>
<a class="sourceLine" id="cb21-3" data-line-number="3"><span class="ex">module</span> load R/3.5.2</a>
<a class="sourceLine" id="cb21-4" data-line-number="4"></a>
<a class="sourceLine" id="cb21-5" data-line-number="5"><span class="va">genome_fasta=</span>reference_genome_hg19.fasta</a>
<a class="sourceLine" id="cb21-6" data-line-number="6"></a>
<a class="sourceLine" id="cb21-7" data-line-number="7"><span class="ex">Rscript</span> ./functions/gatk_process_pipe.R \</a>
<a class="sourceLine" id="cb21-8" data-line-number="8">--reference_fasta <span class="va">$genome_fasta</span> \</a>
<a class="sourceLine" id="cb21-9" data-line-number="9">--bamfile data/test-fastq/star-pass2/SRX729580Aligned.sortedByCoord.out.RG.bam \</a>
<a class="sourceLine" id="cb21-10" data-line-number="10">--sampleName SRX729580 \</a>
<a class="sourceLine" id="cb21-11" data-line-number="11">--knownSites1 path_to_GATK_Bundle_files/dbsnp_138.hg19.excluding_sites_after_129.vcf \</a>
<a class="sourceLine" id="cb21-12" data-line-number="12">--knownSites2 path_to_GATK_Bundle_files/Mills_and_1000G_gold_standard.indels.hg19.sites.vcf \</a>
<a class="sourceLine" id="cb21-13" data-line-number="13">--knownSites3 path_to_GATK_Bundle_files/1000G_phase1.indels.hg19.sites.vcf </a></code></pre></div>
<p>The function above is a wrapper for the following <code>GATK3</code> calls.</p>
<div id="splitncigarreads" class="section level3">
<h3><span class="header-section-number">3.2.1</span> SplitNCigarReads</h3>
<div class="sourceCode" id="cb22"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb22-1" data-line-number="1"><span class="ex">gatk</span> -T SplitNCigarReads -R path_to_genome.fa \</a>
<a class="sourceLine" id="cb22-2" data-line-number="2">-I data/test-fastq/star-pass2/SRX729580Aligned.sortedByCoord.out.RG.bam \</a>
<a class="sourceLine" id="cb22-3" data-line-number="3">-o data/test-fastq/star-pass2/SRX729580Aligned.reorderedDupl.rg.split.bam \</a>
<a class="sourceLine" id="cb22-4" data-line-number="4">--filter_mismatching_base_and_quals -U ALLOW_N_CIGAR_READS -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 \</a>
<a class="sourceLine" id="cb22-5" data-line-number="5">--log_to_file data/test-fastq/star-pass2/SRX729580_RG_DUPL_SPLIT_log</a></code></pre></div>
</div>
<div id="base-recalibration" class="section level3">
<h3><span class="header-section-number">3.2.2</span> Base recalibration</h3>
<p>Base recalibration using known sites downloaded from the <a href="https://github.com/snewhouse/ngs_nextflow/wiki/GATK-Bundle">GATK Bundle</a>. More information about base recalibration can be found on <a href="https://software.broadinstitute.org/gatk/documentation/tooldocs/3.8-0/org_broadinstitute_gatk_tools_walkers_bqsr_BaseRecalibrator.php">GATK website</a>.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb23-1" data-line-number="1"><span class="ex">module</span> load gatk/3.7.0</a>
<a class="sourceLine" id="cb23-2" data-line-number="2"></a>
<a class="sourceLine" id="cb23-3" data-line-number="3"><span class="ex">gatk</span> -T BaseRecalibrator -R path_to_genome.fa \</a>
<a class="sourceLine" id="cb23-4" data-line-number="4">-I data/test-fastq/star-pass2/SRX729580Aligned.reorderedDupl.rg.split.bam -nct 8 \</a>
<a class="sourceLine" id="cb23-5" data-line-number="5">-knownSites path_to_GATK_Bundle_files/dbsnp_138.hg19.excluding_sites_after_129.vcf \</a>
<a class="sourceLine" id="cb23-6" data-line-number="6">-knownSites path_to_GATK_Bundle_files/Mills_and_1000G_gold_standard.indels.hg19.sites.vcf \</a>
<a class="sourceLine" id="cb23-7" data-line-number="7">-knownSites path_to_GATK_Bundle_files/1000G_phase1.indels.hg19.sites.vcf \</a>
<a class="sourceLine" id="cb23-8" data-line-number="8">-o data/test-fastq/star-pass2/BaseQRecal/SRX729580/SRX729580_recal_data.table \</a>
<a class="sourceLine" id="cb23-9" data-line-number="9">--log_to_file data/test-fastq/star-pass2/BaseQRecal/SRX729580/SRX729580_recal_step1_log </a>
<a class="sourceLine" id="cb23-10" data-line-number="10"></a>
<a class="sourceLine" id="cb23-11" data-line-number="11"><span class="ex">gatk</span> -T BaseRecalibrator -R path_hg19_reference/genome.fa \</a>
<a class="sourceLine" id="cb23-12" data-line-number="12">-I data/test-fastq/star-pass2/SRX729580Aligned.reorderedDupl.rg.split.bam -nct 8 \</a>
<a class="sourceLine" id="cb23-13" data-line-number="13">-knownSites path_to_GATK_Bundle_files/dbsnp_138.hg19.excluding_sites_after_129.vcf \</a>
<a class="sourceLine" id="cb23-14" data-line-number="14">-knownSites path_to_GATK_Bundle_files/Mills_and_1000G_gold_standard.indels.hg19.sites.vcf \</a>
<a class="sourceLine" id="cb23-15" data-line-number="15">-knownSites path_to_GATK_Bundle_files/1000G_phase1.indels.hg19.sites.vcf \</a>
<a class="sourceLine" id="cb23-16" data-line-number="16">-BQSR data/test-fastq/star-pass2/BaseQRecal/SRX729580/SampleName_recal_data.table \</a>
<a class="sourceLine" id="cb23-17" data-line-number="17">-o data/test-fastq/star-pass2/BaseQRecal/SRX729580/SRX729580_post_recal_data.table \</a>
<a class="sourceLine" id="cb23-18" data-line-number="18">--log_to_file data/test-fastq/star-pass2/BaseQRecal/SRX729580/SRX729580_recal_step2_log </a>
<a class="sourceLine" id="cb23-19" data-line-number="19"></a>
<a class="sourceLine" id="cb23-20" data-line-number="20"><span class="ex">gatk</span> -T AnalyzeCovariates -R path_hg19_reference/genome.fa \</a>
<a class="sourceLine" id="cb23-21" data-line-number="21">-before data/test-fastq/star-pass2/BaseQRecal/SRX729580/SSRX729580_recal_data.table \</a>
<a class="sourceLine" id="cb23-22" data-line-number="22">-after data/test-fastq/star-pass2/BaseQRecal/SRX729580/SRX729580_post_recal_data.table \</a>
<a class="sourceLine" id="cb23-23" data-line-number="23">-csv data/test-fastq/star-pass2/BaseQRecal/SRX729580/SSRX729580_recalibration_plots.csv \</a>
<a class="sourceLine" id="cb23-24" data-line-number="24">-plots data/test-fastq/star-pass2/BaseQRecal/SRX729580/SRX729580_recalibration_plots.pdf \</a>
<a class="sourceLine" id="cb23-25" data-line-number="25">--log_to_file data/test-fastq/star-pass2/BaseQRecal/SRX729580/SRX729580_recal_analyseCov_log </a>
<a class="sourceLine" id="cb23-26" data-line-number="26"></a>
<a class="sourceLine" id="cb23-27" data-line-number="27"><span class="ex">gatk</span> -T PrintReads -R path_hg19_reference/genome.fa \</a>
<a class="sourceLine" id="cb23-28" data-line-number="28">-I data/test-fastq/star-pass2/SRX729580Aligned.reorderedDupl.rg.split.bam \</a>
<a class="sourceLine" id="cb23-29" data-line-number="29">-o data/test-fastq/star-pass2/SRX729580Recal.reorderedDupl.rg.split.bam \</a>
<a class="sourceLine" id="cb23-30" data-line-number="30">-nct 8 -BQSR data/test-fastq/star-pass2/BaseQRecal/SRX729580/SRX729580_post_recal_data.table \</a>
<a class="sourceLine" id="cb23-31" data-line-number="31">--log_to_file data/test-fastq/star-pass2/BaseQRecal/SRX729580/SRX729580_Log_recalibrated_bases</a></code></pre></div>
<p>MultiQC was used to create summary after running pre-processing with GATK <a href="https://multiqc.info/docs/#gatk" class="uri">https://multiqc.info/docs/#gatk</a>.</p>
</div>
</div>
<div id="ch:call-variants" class="section level2">
<h2><span class="header-section-number">3.3</span> Variant calling</h2>
<p>Variants are called on regions of interest corresponding to the gene bodies of the genes whose variants are reported in <strong>Supplemental Table 3</strong> of <span class="citation">(Lavallée et al. <a href="#ref-Lavallee2016-sf">2016</a>)</span>. The <code>bed</code> file with genomic regions is <code>./data/target_regions.bed</code>.</p>
<p>Variants are called with <code>MuTect2</code>, <code>VarScan</code> and <code>VarDict</code> in tumour-only mode and annotated using the <a href="https://asia.ensembl.org/info/docs/tools/vep/index.html">Variant Effect Preditor (VEP)</a>. The directory needed for variant calling with VarDict (<code>path_to_vardict/VarDict</code>) was downloaded from VarDict GitHub page <a href="https://github.com/AstraZeneca-NGS/VarDict" class="uri">https://github.com/AstraZeneca-NGS/VarDict</a>.</p>
<p>The <code>./functions/call_variants.R</code> function is a wrapper to call and annotate variants with the callers listed above.</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb24-1" data-line-number="1"><span class="ex">module</span> load R/3.5.2 <span class="co"># needed for vardict</span></a>
<a class="sourceLine" id="cb24-2" data-line-number="2"><span class="ex">module</span> load vardict/1.5.1</a>
<a class="sourceLine" id="cb24-3" data-line-number="3"><span class="ex">module</span> load vcftools/0.1.13 <span class="co"># needed for variant parsing</span></a>
<a class="sourceLine" id="cb24-4" data-line-number="4"><span class="ex">module</span> load ensembl-vep/89.0</a>
<a class="sourceLine" id="cb24-5" data-line-number="5"></a>
<a class="sourceLine" id="cb24-6" data-line-number="6"><span class="co"># Vardict tumour-only call</span></a>
<a class="sourceLine" id="cb24-7" data-line-number="7"><span class="ex">Rscript</span> ./functions/call_variants.R \</a>
<a class="sourceLine" id="cb24-8" data-line-number="8">--reference_fasta path_hg19_reference/genome.fa \</a>
<a class="sourceLine" id="cb24-9" data-line-number="9">--bamfile data/test-fastq/star-pass2/SRX729580Aligned.sortedByCoord.out.RG.split.bam \</a>
<a class="sourceLine" id="cb24-10" data-line-number="10">--sampleName SRX729580 \</a>
<a class="sourceLine" id="cb24-11" data-line-number="11">--regions ./data/target_regions.bed \</a>
<a class="sourceLine" id="cb24-12" data-line-number="12">--genome_assembly <span class="st">&#39;GRCh37&#39;</span> \</a>
<a class="sourceLine" id="cb24-13" data-line-number="13">--VarDict_dir path_to_vardict_directory \</a>
<a class="sourceLine" id="cb24-14" data-line-number="14">--caller vardict \</a>
<a class="sourceLine" id="cb24-15" data-line-number="15">--outputdir_suffix <span class="st">&quot;caller_defaults&quot;</span> \</a>
<a class="sourceLine" id="cb24-16" data-line-number="16">--VEPcall <span class="st">&#39;vep --dir_cache path_to_vep_cache/.vep --offline&#39;</span> \</a>
<a class="sourceLine" id="cb24-17" data-line-number="17">--output_directory ./results/</a>
<a class="sourceLine" id="cb24-18" data-line-number="18"> </a>
<a class="sourceLine" id="cb24-19" data-line-number="19"><span class="co"># VarScan tumour-only call </span></a>
<a class="sourceLine" id="cb24-20" data-line-number="20"><span class="ex">module</span> load R/3.5.2</a>
<a class="sourceLine" id="cb24-21" data-line-number="21"><span class="ex">module</span> load varscan/2.3.9</a>
<a class="sourceLine" id="cb24-22" data-line-number="22"><span class="ex">module</span> load vcftools/0.1.13 <span class="co"># needed for variant parsing</span></a>
<a class="sourceLine" id="cb24-23" data-line-number="23"><span class="ex">module</span> load ensembl-vep/89.0</a>
<a class="sourceLine" id="cb24-24" data-line-number="24"></a>
<a class="sourceLine" id="cb24-25" data-line-number="25"><span class="ex">Rscript</span> ./functions/call_variants.R \</a>
<a class="sourceLine" id="cb24-26" data-line-number="26">--reference_fasta path_to_genome.fa \</a>
<a class="sourceLine" id="cb24-27" data-line-number="27">--bamfile data/test-fastq/star-pass2/SRX729580Recal.reorderedDupl.rg.split.bam \</a>
<a class="sourceLine" id="cb24-28" data-line-number="28">--sampleName SRX729580 \</a>
<a class="sourceLine" id="cb24-29" data-line-number="29">--regions ./data/target_regions.bed \</a>
<a class="sourceLine" id="cb24-30" data-line-number="30">--genome_assembly <span class="st">&#39;GRCh37&#39;</span> \</a>
<a class="sourceLine" id="cb24-31" data-line-number="31">--caller varscan \</a>
<a class="sourceLine" id="cb24-32" data-line-number="32">--outputdir_suffix <span class="st">&quot;caller_defaults&quot;</span> \</a>
<a class="sourceLine" id="cb24-33" data-line-number="33">--VEPcall <span class="st">&#39;vep --dir_cache path_to_vep_cache/.vep --offline&#39;</span> \</a>
<a class="sourceLine" id="cb24-34" data-line-number="34">--output_directory ./results/</a>
<a class="sourceLine" id="cb24-35" data-line-number="35"></a>
<a class="sourceLine" id="cb24-36" data-line-number="36"></a>
<a class="sourceLine" id="cb24-37" data-line-number="37"><span class="co"># MuTect tumour-only call </span></a>
<a class="sourceLine" id="cb24-38" data-line-number="38"><span class="ex">module</span> load R/3.5.2</a>
<a class="sourceLine" id="cb24-39" data-line-number="39"><span class="ex">module</span> load gatk/3.7.0 </a>
<a class="sourceLine" id="cb24-40" data-line-number="40"><span class="ex">module</span> load vcftools/0.1.13</a>
<a class="sourceLine" id="cb24-41" data-line-number="41"><span class="ex">module</span> load ensembl-vep/89.0</a>
<a class="sourceLine" id="cb24-42" data-line-number="42"></a>
<a class="sourceLine" id="cb24-43" data-line-number="43"><span class="ex">Rscript</span> ./functions/call_variants.R \</a>
<a class="sourceLine" id="cb24-44" data-line-number="44">--reference_fasta path_to_genome.fa \</a>
<a class="sourceLine" id="cb24-45" data-line-number="45">--bamfile data/test-fastq/star-pass2/SRX729580Recal.reorderedDupl.rg.split.bam \</a>
<a class="sourceLine" id="cb24-46" data-line-number="46">--sampleName SRX729580 \</a>
<a class="sourceLine" id="cb24-47" data-line-number="47">--regions ./data/target_regions.bed \</a>
<a class="sourceLine" id="cb24-48" data-line-number="48">--genome_assembly <span class="st">&#39;GRCh37&#39;</span> \</a>
<a class="sourceLine" id="cb24-49" data-line-number="49">--caller mutect \</a>
<a class="sourceLine" id="cb24-50" data-line-number="50">--outputdir_suffix <span class="st">&quot;caller_defaults&quot;</span> \</a>
<a class="sourceLine" id="cb24-51" data-line-number="51">--VEPcall <span class="st">&#39;vep --dir_cache path_to_vep_cache/.vep --offline&#39;</span> \</a>
<a class="sourceLine" id="cb24-52" data-line-number="52">--output_directory ./results/</a></code></pre></div>
<p>The calls using the function above are wrappers for the calls done by each caller using the settings below.</p>
<div id="mutect2-settings" class="section level3">
<h3><span class="header-section-number">3.3.1</span> MuTect2 settings</h3>
<div class="sourceCode" id="cb25"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb25-1" data-line-number="1"><span class="ex">gatk</span> -T MuTect2 -R path_to_genome.fa \</a>
<a class="sourceLine" id="cb25-2" data-line-number="2">-I:tumor data/test-fastq/star-pass2/SRX729580Recal.reorderedDupl.rg.split.bam \</a>
<a class="sourceLine" id="cb25-3" data-line-number="3">-L ./data/target_regions.bed \</a>
<a class="sourceLine" id="cb25-4" data-line-number="4">-o ./results/mutect/regions_caller_defaults/SRX729580_germline_snvs_indels.vcf \</a>
<a class="sourceLine" id="cb25-5" data-line-number="5">-log ./results/mutect/regions_caller_defaults/SRX729580_germline_snvs_indels_log</a></code></pre></div>
</div>
<div id="samtools-varscan2-settings" class="section level3">
<h3><span class="header-section-number">3.3.2</span> Samtools + VarScan2 settings</h3>
<div class="sourceCode" id="cb26"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb26-1" data-line-number="1"><span class="ex">samtools</span> mpileup --output-tags AD,ADF,ADR,DP,SP \</a>
<a class="sourceLine" id="cb26-2" data-line-number="2">--fasta-ref path_to_genome.fa \</a>
<a class="sourceLine" id="cb26-3" data-line-number="3">-l ./data/target_regions.bed data/test-fastq/star-pass2/SSRX729580Aligned.sortedByCoord.out.RG.split.recalibrated.bam <span class="kw">|</span> <span class="ex">varscan</span> mpileup2cns --variants 1 --output-vcf 1 --min-var-freq 0.01 <span class="op">&gt;</span> ./results/varscan/regions_caller_defaults/SSRX729580_germline_snvs_indels.vcf <span class="op">2&gt;</span> ./results/varscan/regions_caller_defaults/SSRX729580_germline_snvs_indels_log</a></code></pre></div>
</div>
<div id="vardict-settings" class="section level3">
<h3><span class="header-section-number">3.3.3</span> VarDict settings</h3>
<div class="sourceCode" id="cb27"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb27-1" data-line-number="1"><span class="ex">vardict</span> -f 0.05 -c 1 -S 2 -E 3 -g 4 -r 2 -t -th 10 -v -G  path_to_genome.fa \</a>
<a class="sourceLine" id="cb27-2" data-line-number="2">-b data/test-fastq/star-pass2/SRX729580Aligned.sortedByCoord.out.RG.split.bam ./data/target_regions.bed  <span class="kw">|</span> <span class="ex">vardict_dir/teststrandbias.R</span> <span class="kw">|</span> <span class="ex">vardict_dir/var2vcf_valid.pl</span> -N -E -f 0.05 <span class="op">&gt;</span>  ./results/vardict/regions_default-settings/SSRX729580_germline_snvs_indels.vcf</a></code></pre></div>
</div>
<div id="vep-settings" class="section level3">
<h3><span class="header-section-number">3.3.4</span> VEP settings</h3>
<div class="sourceCode" id="cb28"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb28-1" data-line-number="1"><span class="ex">module</span> load ensembl-vep/89.0</a>
<a class="sourceLine" id="cb28-2" data-line-number="2"></a>
<a class="sourceLine" id="cb28-3" data-line-number="3"><span class="ex">vep</span> --dir_cache dir_to_VEP_cache/.vep --offline \</a>
<a class="sourceLine" id="cb28-4" data-line-number="4">-i ./results/vardict/regions_default-settings/SSRX729580_germline_snvs_indels.vcf \</a>
<a class="sourceLine" id="cb28-5" data-line-number="5">-o ./results/vardict/regions_default-settings/annotated_variants/SSRX729580_germline_annotated.vcf \</a>
<a class="sourceLine" id="cb28-6" data-line-number="6">--cache --everything --force_overwrite --assembly GRCh37 --fork 12 --vcf --port 3337</a></code></pre></div>
</div>
</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-Lavallee2016-sf">
<p>Lavallée, Vincent-Philippe, Sébastien Lemieux, Geneviève Boucher, Patrick Gendron, Isabel Boivin, Richard Neil Armstrong, Guy Sauvageau, and Josée Hébert. 2016. “RNA-sequencing Analysis of Core Binding Factor AML Identifies Recurrent ZBTB7A Mutations and Defines RUNX1-CBFA2T3 Fusion Signature.” <em>Blood</em>, March.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="ch-leucegene-down.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="standardise-variants-output.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
