<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>2 Download, quality control and downsample the Leucegene RNA-Seq samples | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq</title>
  <meta name="description" content="2 Download, quality control and downsample the Leucegene RNA-Seq samples | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  <meta name="generator" content="bookdown 0.16 and GitBook 2.6.7" />

  <meta property="og:title" content="2 Download, quality control and downsample the Leucegene RNA-Seq samples | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="2 Download, quality control and downsample the Leucegene RNA-Seq samples | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  
  
  

<meta name="author" content="Anna Quaglieri" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="ch-acs.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Aim of this document</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#setup"><i class="fa fa-check"></i><b>1.1</b> Setup</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#software-used"><i class="fa fa-check"></i><b>1.2</b> Software used</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html"><i class="fa fa-check"></i><b>2</b> Download, quality control and downsample the Leucegene RNA-Seq samples</a><ul>
<li class="chapter" data-level="2.1" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#download-rna-seq-data-from-geo"><i class="fa fa-check"></i><b>2.1</b> Download RNA-Seq data from GEO</a><ul>
<li class="chapter" data-level="2.1.1" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#get-srx-sample-names"><i class="fa fa-check"></i><b>2.1.1</b> Get SRX sample names</a></li>
<li class="chapter" data-level="2.1.2" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#create-ncbi-query"><i class="fa fa-check"></i><b>2.1.2</b> Create NCBI query</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#ch:leucegene-downsample"><i class="fa fa-check"></i><b>2.2</b> Downsampling FASTQ files to a fixed number of reads</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch-acs.html"><a href="ch-acs.html"><i class="fa fa-check"></i><b>3</b> Align, call and standardise</a><ul>
<li class="chapter" data-level="3.1" data-path="ch-acs.html"><a href="ch-acs.html#ch:align"><i class="fa fa-check"></i><b>3.1</b> Alignement, Read Groups, Mark duplicates</a><ul>
<li class="chapter" data-level="3.1.1" data-path="ch-acs.html"><a href="ch-acs.html#star-index"><i class="fa fa-check"></i><b>3.1.1</b> STAR index</a></li>
<li class="chapter" data-level="3.1.2" data-path="ch-acs.html"><a href="ch-acs.html#star-1pass"><i class="fa fa-check"></i><b>3.1.2</b> STAR-1pass</a></li>
<li class="chapter" data-level="3.1.3" data-path="ch-acs.html"><a href="ch-acs.html#star-2pass"><i class="fa fa-check"></i><b>3.1.3</b> STAR-2pass</a></li>
<li class="chapter" data-level="3.1.4" data-path="ch-acs.html"><a href="ch-acs.html#sec:post-align"><i class="fa fa-check"></i><b>3.1.4</b> Processing post alignment</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="ch-acs.html"><a href="ch-acs.html#gatk-pre-processing-to-call-variants-with-mutect-and-varscan"><i class="fa fa-check"></i><b>3.2</b> GATK pre-processing to call variants with MuTect and VarScan</a><ul>
<li class="chapter" data-level="3.2.1" data-path="ch-acs.html"><a href="ch-acs.html#splitncigarreads"><i class="fa fa-check"></i><b>3.2.1</b> SplitNCigarReads</a></li>
<li class="chapter" data-level="3.2.2" data-path="ch-acs.html"><a href="ch-acs.html#base-recalibration"><i class="fa fa-check"></i><b>3.2.2</b> Base recalibration</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="ch-acs.html"><a href="ch-acs.html#ch:call-variants"><i class="fa fa-check"></i><b>3.3</b> Variant calling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="ch-acs.html"><a href="ch-acs.html#mutect2-settings"><i class="fa fa-check"></i><b>3.3.1</b> MuTect2 settings</a></li>
<li class="chapter" data-level="3.3.2" data-path="ch-acs.html"><a href="ch-acs.html#samtools-varscan2-settings"><i class="fa fa-check"></i><b>3.3.2</b> Samtools + VarScan2 settings</a></li>
<li class="chapter" data-level="3.3.3" data-path="ch-acs.html"><a href="ch-acs.html#vardict-settings"><i class="fa fa-check"></i><b>3.3.3</b> VarDict settings</a></li>
<li class="chapter" data-level="3.3.4" data-path="ch-acs.html"><a href="ch-acs.html#vep-settings"><i class="fa fa-check"></i><b>3.3.4</b> VEP settings</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="ch-acs.html"><a href="ch-acs.html#standardise-variants-output"><i class="fa fa-check"></i><b>3.4</b> Standardise variants output</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch-tcga-down.html"><a href="ch-tcga-down.html"><i class="fa fa-check"></i><b>4</b> TCGA-LAML downsampling</a></li>
<li class="chapter" data-level="5" data-path="ch-deploy.html"><a href="ch-deploy.html"><i class="fa fa-check"></i><b>5</b> Gitbook publication and deployment</a></li>
<li class="chapter" data-level="6" data-path="ch-refer.html"><a href="ch-refer.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supporting scripts: Finding a suitable library size to call variants in RNA-Seq</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch:leucegene-down" class="section level1">
<h1><span class="header-section-number">2</span> Download, quality control and downsample the Leucegene RNA-Seq samples</h1>
<div id="download-rna-seq-data-from-geo" class="section level2">
<h2><span class="header-section-number">2.1</span> Download RNA-Seq data from GEO</h2>
<p>The initial unaligned RNA-Seq CBF-AML samples are downloaded from <a href="https://www.ncbi.nlm.nih.gov/geo/">GEO</a> using the accession numbers GSE49642, GSE52656, GSE62190, GSE66917, and GSE67039. Only 46 FASTQ files of samples whose mutations are reported in Supplemental Table 3 of <span class="citation">(Lavallée et al. <a href="#ref-Lavallee2016-sf">2016</a>)</span> are kept for the analysis. Sample SRX381851 is removed due to low number of available reads. The file <code>data/CBF_AML_samples.csv</code> contains the sample names of the 46 samples used in the study.</p>
<p>Below is an example using accession number GSE52656 where one of the CBF-AML samples is present, namely 07H099.</p>
<div id="get-srx-sample-names" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Get SRX sample names</h3>
<p>The <code>GEOquery</code> package can be used to extract <code>SRX</code> files linked to an accession number.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="co"># CRAN packages</span></a>
<a class="sourceLine" id="cb2-2" data-line-number="2"><span class="kw">library</span>(tidyverse)</a>
<a class="sourceLine" id="cb2-3" data-line-number="3"><span class="kw">library</span>(knitr)</a>
<a class="sourceLine" id="cb2-4" data-line-number="4"><span class="kw">library</span>(here)</a>
<a class="sourceLine" id="cb2-5" data-line-number="5"></a>
<a class="sourceLine" id="cb2-6" data-line-number="6"><span class="co"># Bioconductor packages</span></a>
<a class="sourceLine" id="cb2-7" data-line-number="7"><span class="cf">if</span> (<span class="op">!</span><span class="kw">requireNamespace</span>(<span class="st">&quot;BiocManager&quot;</span>, <span class="dt">quietly =</span> <span class="ot">TRUE</span>))</a>
<a class="sourceLine" id="cb2-8" data-line-number="8">    <span class="kw">install.packages</span>(<span class="st">&quot;BiocManager&quot;</span>)</a>
<a class="sourceLine" id="cb2-9" data-line-number="9">BiocManager<span class="op">::</span><span class="kw">install</span>(<span class="st">&quot;GEOquery&quot;</span>,<span class="dt">update =</span> <span class="ot">FALSE</span>)</a></code></pre></div>
<pre><code>## 
## The downloaded binary packages are in
##  /var/folders/2b/9q8hw_092dg9d80pbmfjvndr00023z/T//Rtmpy6YArq/downloaded_packages</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1"><span class="kw">library</span>(GEOquery)</a></code></pre></div>
<p>Below the accession number <strong>GSE52656</strong> is used as example.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" data-line-number="1"><span class="co"># Get matrix files for every accession number</span></a>
<a class="sourceLine" id="cb5-2" data-line-number="2">series_matrix_info &lt;-<span class="st"> </span><span class="cf">function</span>(gse){</a>
<a class="sourceLine" id="cb5-3" data-line-number="3">gsed &lt;-<span class="st"> </span><span class="kw">getGEO</span>(gse,<span class="dt">GSEMatrix=</span><span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb5-4" data-line-number="4">gse.mat &lt;-<span class="st"> </span><span class="kw">pData</span>(<span class="kw">phenoData</span>(gsed[[<span class="dv">1</span>]]))</a>
<a class="sourceLine" id="cb5-5" data-line-number="5">reduced &lt;-<span class="st"> </span>gse.mat[,<span class="kw">c</span>(<span class="st">&quot;title&quot;</span>,<span class="st">&quot;geo_accession&quot;</span>,<span class="st">&quot;relation.1&quot;</span>)]</a>
<a class="sourceLine" id="cb5-6" data-line-number="6"><span class="kw">write.csv</span>(reduced,<span class="kw">file.path</span>(<span class="kw">here</span>(),<span class="st">&quot;data&quot;</span>,<span class="kw">paste</span>(gse,<span class="st">&quot;_&quot;</span>,<span class="kw">nrow</span>(gse.mat),<span class="st">&quot;.csv&quot;</span>,<span class="dt">sep=</span><span class="st">&quot;&quot;</span>)),<span class="dt">row.names =</span> <span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb5-7" data-line-number="7">}</a>
<a class="sourceLine" id="cb5-8" data-line-number="8"></a>
<a class="sourceLine" id="cb5-9" data-line-number="9"><span class="kw">series_matrix_info</span>(<span class="st">&quot;GSE62190&quot;</span>) </a></code></pre></div>
<p>Every row in Table <a href="ch-leucegene-down.html#tab:SRX">2.1</a> contains sample names (<code>title</code>) and <code>GSM</code> numbers. In order to download a particular sample we need the <code>SRA</code> terms which are the names starting with: <code>SRX***</code> in the <code>relation.1</code> column.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1">matrix_file &lt;-<span class="st"> </span><span class="kw">list.files</span>(<span class="dt">path =</span> <span class="kw">file.path</span>(<span class="kw">here</span>(),<span class="st">&quot;data&quot;</span>),<span class="dt">pattern =</span> <span class="st">&quot;GSE&quot;</span>,<span class="dt">full.names =</span> <span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb6-2" data-line-number="2">GSEmatrix &lt;-<span class="st"> </span><span class="kw">read_csv</span>(matrix_file)</a>
<a class="sourceLine" id="cb6-3" data-line-number="3"></a>
<a class="sourceLine" id="cb6-4" data-line-number="4"><span class="kw">kable</span>(GSEmatrix[<span class="dv">1</span><span class="op">:</span><span class="dv">5</span>,],<span class="dt">caption=</span><span class="st">&quot;SRX sample names linked to the accession number GSE62190&quot;</span>)</a></code></pre></div>
<table>
<caption><span id="tab:SRX">Table 2.1: </span>SRX sample names linked to the accession number GSE62190</caption>
<thead>
<tr class="header">
<th align="left">title</th>
<th align="left">geo_accession</th>
<th align="left">relation.1</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">02H017</td>
<td align="left">GSM1521543</td>
<td align="left">SRA: <a href="https://www.ncbi.nlm.nih.gov/sra?term=SRX729552" class="uri">https://www.ncbi.nlm.nih.gov/sra?term=SRX729552</a></td>
</tr>
<tr class="even">
<td align="left">06H152</td>
<td align="left">GSM1521544</td>
<td align="left">SRA: <a href="https://www.ncbi.nlm.nih.gov/sra?term=SRX729553" class="uri">https://www.ncbi.nlm.nih.gov/sra?term=SRX729553</a></td>
</tr>
<tr class="odd">
<td align="left">08H085</td>
<td align="left">GSM1521545</td>
<td align="left">SRA: <a href="https://www.ncbi.nlm.nih.gov/sra?term=SRX729554" class="uri">https://www.ncbi.nlm.nih.gov/sra?term=SRX729554</a></td>
</tr>
<tr class="even">
<td align="left">08H129</td>
<td align="left">GSM1521546</td>
<td align="left">SRA: <a href="https://www.ncbi.nlm.nih.gov/sra?term=SRX729555" class="uri">https://www.ncbi.nlm.nih.gov/sra?term=SRX729555</a></td>
</tr>
<tr class="odd">
<td align="left">12H057</td>
<td align="left">GSM1521547</td>
<td align="left">SRA: <a href="https://www.ncbi.nlm.nih.gov/sra?term=SRX729556" class="uri">https://www.ncbi.nlm.nih.gov/sra?term=SRX729556</a></td>
</tr>
</tbody>
</table>
<p>With some string processing we can extract the <code>SRX</code> entries.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" data-line-number="1">GSEmatrix<span class="op">$</span>SRX &lt;-<span class="st"> </span>stringr<span class="op">::</span><span class="kw">str_extract</span>(<span class="dt">string =</span> GSEmatrix<span class="op">$</span>relation<span class="fl">.1</span>,<span class="dt">pattern =</span> <span class="st">&quot;SRX[0-9][0-9][0-9][0-9][0-9][0-9]&quot;</span>)</a>
<a class="sourceLine" id="cb7-2" data-line-number="2">GSEmatrix<span class="op">$</span>relation<span class="fl">.1</span> &lt;-<span class="st"> </span><span class="ot">NULL</span></a>
<a class="sourceLine" id="cb7-3" data-line-number="3"><span class="kw">kable</span>(<span class="kw">head</span>(GSEmatrix))</a></code></pre></div>
<table>
<thead>
<tr class="header">
<th align="left">title</th>
<th align="left">geo_accession</th>
<th align="left">SRX</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">02H017</td>
<td align="left">GSM1521543</td>
<td align="left">SRX729552</td>
</tr>
<tr class="even">
<td align="left">06H152</td>
<td align="left">GSM1521544</td>
<td align="left">SRX729553</td>
</tr>
<tr class="odd">
<td align="left">08H085</td>
<td align="left">GSM1521545</td>
<td align="left">SRX729554</td>
</tr>
<tr class="even">
<td align="left">08H129</td>
<td align="left">GSM1521546</td>
<td align="left">SRX729555</td>
</tr>
<tr class="odd">
<td align="left">12H057</td>
<td align="left">GSM1521547</td>
<td align="left">SRX729556</td>
</tr>
<tr class="even">
<td align="left">01H001</td>
<td align="left">GSM1521548</td>
<td align="left">SRX729557</td>
</tr>
</tbody>
</table>
<p>The next chunk extracts only samples whose variants were published in <span class="citation">(Lavallée et al. <a href="#ref-Lavallee2016-sf">2016</a>)</span>.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" data-line-number="1"><span class="co"># CBF-AML Sample</span></a>
<a class="sourceLine" id="cb8-2" data-line-number="2">CBF_AML_info &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="kw">file.path</span>(<span class="kw">here</span>(),<span class="st">&quot;data/CBF_AML_samples.csv&quot;</span>))</a>
<a class="sourceLine" id="cb8-3" data-line-number="3">cbf.samples &lt;-<span class="st"> </span>GSEmatrix[GSEmatrix<span class="op">$</span>title <span class="op">%in%</span><span class="st"> </span>CBF_AML_info<span class="op">$</span>title, ]</a></code></pre></div>
</div>
<div id="create-ncbi-query" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Create NCBI query</h3>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb9-1" data-line-number="1">search_ncbi &lt;-<span class="st"> </span><span class="kw">paste</span>(cbf.samples<span class="op">$</span>SRX,<span class="dt">collapse=</span><span class="st">&quot; OR &quot;</span>)</a>
<a class="sourceLine" id="cb9-2" data-line-number="2">search_ncbi</a></code></pre></div>
<pre><code>## [1] &quot;SRX729580 OR SRX729581 OR SRX729582 OR SRX729583 OR SRX729584 OR SRX729585 OR SRX729586 OR SRX729587 OR SRX729588 OR SRX729589 OR SRX729590 OR SRX729591 OR SRX729592 OR SRX729602 OR SRX729603 OR SRX729604 OR SRX729605 OR SRX729606 OR SRX729607 OR SRX729608 OR SRX729609 OR SRX729610 OR SRX729611 OR SRX729612 OR SRX729613 OR SRX729615 OR SRX729616 OR SRX729617 OR SRX729618 OR SRX729619 OR SRX729620 OR SRX729621 OR SRX729622 OR SRX729623 OR SRX729624 OR SRX729625 OR SRX729626 OR SRX729627 OR SRX729628 OR SRX729629 OR SRX729630 OR SRX729631 OR SRX729632 OR SRX729633&quot;</code></pre>
<p>The search SRX729580 OR SRX729581 OR SRX729582 OR SRX729583 OR SRX729584 OR SRX729585 OR SRX729586 OR SRX729587 OR SRX729588 OR SRX729589 OR SRX729590 OR SRX729591 OR SRX729592 OR SRX729602 OR SRX729603 OR SRX729604 OR SRX729605 OR SRX729606 OR SRX729607 OR SRX729608 OR SRX729609 OR SRX729610 OR SRX729611 OR SRX729612 OR SRX729613 OR SRX729615 OR SRX729616 OR SRX729617 OR SRX729618 OR SRX729619 OR SRX729620 OR SRX729621 OR SRX729622 OR SRX729623 OR SRX729624 OR SRX729625 OR SRX729626 OR SRX729627 OR SRX729628 OR SRX729629 OR SRX729630 OR SRX729631 OR SRX729632 OR SRX729633 can be pasted into NCBI <a href="https://www.ncbi.nlm.nih.gov/sra" class="uri">https://www.ncbi.nlm.nih.gov/sra</a> and by following the intructions in <a href="https://www.ncbi.nlm.nih.gov/sra/docs/sradownload/#download-sequence-data-files-usi" class="uri">https://www.ncbi.nlm.nih.gov/sra/docs/sradownload/#download-sequence-data-files-usi</a> <strong>Download sequence data files using SRA Toolkit</strong> one can download all the <code>SRR</code> run names and information of the runs.</p>
<p>Sample <code>SRX729580</code> is used as example here and it was run across different lanes whose samples are listed in <code>./data/SraAccList.txt</code>.</p>
<p>Six <code>.sra</code> sampels are downloaded with <code>prefetch</code> with the code below that can be run a Unix terminal from the repository directory.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb11-1" data-line-number="1"><span class="ex">module</span> load sra-toolkit/2.8.1</a>
<a class="sourceLine" id="cb11-2" data-line-number="2"><span class="co"># Run in a unix terminal in the repository directory</span></a>
<a class="sourceLine" id="cb11-3" data-line-number="3"><span class="co"># The sra files are downloaded in your home directory under ~/ncbi/public/sra</span></a>
<a class="sourceLine" id="cb11-4" data-line-number="4"><span class="ex">prefetch</span> --option-file ./data/SraAccList.txt</a></code></pre></div>
<p>The SRA files are then converted to <code>FASTQ</code> files with <code>fastq-dump --split-files</code> using <code>fastqc/0.11.8</code>.</p>
<p>The <code>sra</code> files are downloaded by default into your home directory under <code>~/ncbi/public/sra</code>. In the example below I converted the <code>sra</code> to fastq with <code>fastq-dump</code>, performed quality control with <code>fastqc</code> and the FASTQ files to <code>./data/test-fastq</code>.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb12-1" data-line-number="1"><span class="ex">module</span> load fastqc/0.11.8</a>
<a class="sourceLine" id="cb12-2" data-line-number="2"><span class="ex">module</span> load python/3.6.0 <span class="co"># for MultiQC</span></a>
<a class="sourceLine" id="cb12-3" data-line-number="3"><span class="ex">module</span> load parallel</a>
<a class="sourceLine" id="cb12-4" data-line-number="4"></a>
<a class="sourceLine" id="cb12-5" data-line-number="5"><span class="co"># Run in a unix terminal</span></a>
<a class="sourceLine" id="cb12-6" data-line-number="6"><span class="co"># Convert to Fastq</span></a>
<a class="sourceLine" id="cb12-7" data-line-number="7"><span class="ex">fastq-dump</span> --split-files path_to_sra_files/*</a>
<a class="sourceLine" id="cb12-8" data-line-number="8"></a>
<a class="sourceLine" id="cb12-9" data-line-number="9"><span class="co"># Move fastq files in repository directory</span></a>
<a class="sourceLine" id="cb12-10" data-line-number="10"><span class="fu">mkdir</span> ./data/fastq-files</a>
<a class="sourceLine" id="cb12-11" data-line-number="11"><span class="fu">mv</span> path_to_sra_files/*fastq ./data/fastq-files</a>
<a class="sourceLine" id="cb12-12" data-line-number="12"></a>
<a class="sourceLine" id="cb12-13" data-line-number="13"><span class="co"># Multiqc to check quality of files</span></a>
<a class="sourceLine" id="cb12-14" data-line-number="14"><span class="fu">find</span> ./data/fastq-files -name <span class="st">&quot;SRR*.fastq&quot;</span> <span class="op">&gt;</span> ./data/fastq_files.txt</a>
<a class="sourceLine" id="cb12-15" data-line-number="15"><span class="fu">cat</span> ./data/fastq_files.txt <span class="kw">|</span> <span class="ex">parallel</span> -j 10 <span class="st">&quot;fastqc {}&quot;</span></a>
<a class="sourceLine" id="cb12-16" data-line-number="16"></a>
<a class="sourceLine" id="cb12-17" data-line-number="17"><span class="co"># Combine fastqc output and check quality in the html file</span></a>
<a class="sourceLine" id="cb12-18" data-line-number="18"><span class="ex">multiqc</span> ./data/fastq-files/ --interactive -n <span class="st">&quot;files-downloaded&quot;</span> -o ./data/fastq-files/</a>
<a class="sourceLine" id="cb12-19" data-line-number="19"></a>
<a class="sourceLine" id="cb12-20" data-line-number="20"><span class="co"># Move fastqc output into a different directory</span></a>
<a class="sourceLine" id="cb12-21" data-line-number="21"><span class="fu">mkdir</span> ./data/fastqc/</a>
<a class="sourceLine" id="cb12-22" data-line-number="22"><span class="fu">mv</span> ./data/fastq-files/*html  ./data/fastqc/</a>
<a class="sourceLine" id="cb12-23" data-line-number="23"><span class="fu">mv</span> ./data/fastq-files/*zip  ./data/fastqc/</a></code></pre></div>
</div>
</div>
<div id="ch:leucegene-downsample" class="section level2">
<h2><span class="header-section-number">2.2</span> Downsampling FASTQ files to a fixed number of reads</h2>
<p>Each sample was sequenced over multiple lanes and the FASTQ files for one sample were first concatenated and then downsampled. In the example below only the the FASTQ files whose samples are defined in the second argument (<code>./data/sample-to-merge.txt</code>) are combined. The function needs to be run in the directory where the files are saved.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb13-1" data-line-number="1"><span class="bu">cd</span> ./data/fastq-files</a>
<a class="sourceLine" id="cb13-2" data-line-number="2"><span class="ex">python</span> ../../functions/combined_fastqs.py ../match-SRX-SRR.txt ../sample-to-merge.txt</a>
<a class="sourceLine" id="cb13-3" data-line-number="3"></a>
<a class="sourceLine" id="cb13-4" data-line-number="4"><span class="ex">fastqc</span> SRX729580_combined_1.fastq</a>
<a class="sourceLine" id="cb13-5" data-line-number="5"><span class="ex">fastqc</span> SRX729580_combined_2.fastq</a>
<a class="sourceLine" id="cb13-6" data-line-number="6"></a>
<a class="sourceLine" id="cb13-7" data-line-number="7"><span class="ex">multiqc</span> . --interactive -n <span class="st">&quot;files-downloaded-merged&quot;</span> -o .</a></code></pre></div>
<p>The <a href="https://github.com/lh3/seqtk">seqtk</a> tool was then used to downsample an exact number of reads from paired end (PE) FASTQ files. In the code below we downsample the fastq file to 30M fragments and perform QC to check the final number of reads in each file.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb14-1" data-line-number="1"><span class="ex">path-to-seqtk-folder/seqtk</span> sample -s100 SRX729580_combined_1.fastq 30000000 <span class="op">&gt;</span> sub30M_SRX729580_combined_1.fastq</a>
<a class="sourceLine" id="cb14-2" data-line-number="2"><span class="ex">path-to-seqtk-folder/seqtk</span> sample -s100 SRX729580_combined_2.fastq 30000000 <span class="op">&gt;</span> sub30M_SRX729580_combined_2.fastq</a>
<a class="sourceLine" id="cb14-3" data-line-number="3"></a>
<a class="sourceLine" id="cb14-4" data-line-number="4"><span class="co"># Check that the downsampled samples have the correct size and that the downsampling didn&#39;t corrupt the files</span></a>
<a class="sourceLine" id="cb14-5" data-line-number="5"><span class="ex">fastqc</span> sub30M_SRX729580_combined_1.fastq</a>
<a class="sourceLine" id="cb14-6" data-line-number="6"><span class="ex">fastqc</span> sub30M_SRX729580_combined_2.fastq</a>
<a class="sourceLine" id="cb14-7" data-line-number="7"><span class="ex">multiqc</span> . --interactive -n <span class="st">&quot;files-downsampled&quot;</span> -o .</a></code></pre></div>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-Lavallee2016-sf">
<p>Lavallée, Vincent-Philippe, Sébastien Lemieux, Geneviève Boucher, Patrick Gendron, Isabel Boivin, Richard Neil Armstrong, Guy Sauvageau, and Josée Hébert. 2016. “RNA-sequencing Analysis of Core Binding Factor AML Identifies Recurrent ZBTB7A Mutations and Defines RUNX1-CBFA2T3 Fusion Signature.” <em>Blood</em>, March.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="ch-acs.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Supporting-libsize-scripts-book.pdf"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
