<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 TCGA-LAML downsampling | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq</title>
  <meta name="description" content="5 TCGA-LAML downsampling | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  <meta name="generator" content="bookdown 0.16 and GitBook 2.6.7" />

  <meta property="og:title" content="5 TCGA-LAML downsampling | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 TCGA-LAML downsampling | Supporting scripts: Finding a suitable library size to call variants in RNA-Seq" />
  
  
  

<meta name="author" content="Anna Quaglieri" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="standardise-variants-output.html"/>
<link rel="next" href="ch-refer.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Setup</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#software-used"><i class="fa fa-check"></i><b>1.1</b> Software used</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html"><i class="fa fa-check"></i><b>2</b> Download, quality control and downsample the Leucegene RNA-Seq samples</a><ul>
<li class="chapter" data-level="2.1" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#download-rna-seq-data-from-geo"><i class="fa fa-check"></i><b>2.1</b> Download RNA-Seq data from GEO</a><ul>
<li class="chapter" data-level="2.1.1" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#get-srx-sample-names"><i class="fa fa-check"></i><b>2.1.1</b> Get SRX sample names</a></li>
<li class="chapter" data-level="2.1.2" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#create-ncbi-query"><i class="fa fa-check"></i><b>2.1.2</b> Create NCBI query</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="ch-leucegene-down.html"><a href="ch-leucegene-down.html#ch:leucegene-downsample"><i class="fa fa-check"></i><b>2.2</b> Downsampling FASTQ files to a fixed number of reads</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch-acs.html"><a href="ch-acs.html"><i class="fa fa-check"></i><b>3</b> Align, call and standardise</a><ul>
<li class="chapter" data-level="3.1" data-path="ch-acs.html"><a href="ch-acs.html#ch:align"><i class="fa fa-check"></i><b>3.1</b> Alignement, Read Groups, Mark duplicates</a><ul>
<li class="chapter" data-level="3.1.1" data-path="ch-acs.html"><a href="ch-acs.html#star-index"><i class="fa fa-check"></i><b>3.1.1</b> STAR index</a></li>
<li class="chapter" data-level="3.1.2" data-path="ch-acs.html"><a href="ch-acs.html#star-1pass"><i class="fa fa-check"></i><b>3.1.2</b> STAR-1pass</a></li>
<li class="chapter" data-level="3.1.3" data-path="ch-acs.html"><a href="ch-acs.html#star-2pass"><i class="fa fa-check"></i><b>3.1.3</b> STAR-2pass</a></li>
<li class="chapter" data-level="3.1.4" data-path="ch-acs.html"><a href="ch-acs.html#sec:post-align"><i class="fa fa-check"></i><b>3.1.4</b> Processing post alingment</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="ch-acs.html"><a href="ch-acs.html#gatk-pre-processing-to-call-variants-with-mutect-and-varscan"><i class="fa fa-check"></i><b>3.2</b> GATK pre-processing to call variants with MuTect and VarScan</a><ul>
<li class="chapter" data-level="3.2.1" data-path="ch-acs.html"><a href="ch-acs.html#splitncigarreads"><i class="fa fa-check"></i><b>3.2.1</b> SplitNCigarReads</a></li>
<li class="chapter" data-level="3.2.2" data-path="ch-acs.html"><a href="ch-acs.html#base-recalibration"><i class="fa fa-check"></i><b>3.2.2</b> Base recalibration</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="ch-acs.html"><a href="ch-acs.html#ch:call-variants"><i class="fa fa-check"></i><b>3.3</b> Variant calling</a><ul>
<li class="chapter" data-level="3.3.1" data-path="ch-acs.html"><a href="ch-acs.html#mutect2-settings"><i class="fa fa-check"></i><b>3.3.1</b> MuTect2 settings</a></li>
<li class="chapter" data-level="3.3.2" data-path="ch-acs.html"><a href="ch-acs.html#samtools-varscan2-settings"><i class="fa fa-check"></i><b>3.3.2</b> Samtools + VarScan2 settings</a></li>
<li class="chapter" data-level="3.3.3" data-path="ch-acs.html"><a href="ch-acs.html#vardict-settings"><i class="fa fa-check"></i><b>3.3.3</b> VarDict settings</a></li>
<li class="chapter" data-level="3.3.4" data-path="ch-acs.html"><a href="ch-acs.html#vep-settings"><i class="fa fa-check"></i><b>3.3.4</b> VEP settings</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="standardise-variants-output.html"><a href="standardise-variants-output.html"><i class="fa fa-check"></i><b>4</b> Standardise variants output</a></li>
<li class="chapter" data-level="5" data-path="ch-tcga-down.html"><a href="ch-tcga-down.html"><i class="fa fa-check"></i><b>5</b> TCGA-LAML downsampling</a></li>
<li class="chapter" data-level="6" data-path="ch-refer.html"><a href="ch-refer.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Supporting scripts: Finding a suitable library size to call variants in RNA-Seq</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch:tcga-down" class="section level1">
<h1><span class="header-section-number">5</span> TCGA-LAML downsampling</h1>
<p>1.24 is taken as the ratio between the mean proportion of mapped reads in the Leucegene and the TCGA-LAML cohorts. This is discussed in the Methods paper.</p>
<p><code>TCGA-LAML-multiqc_samtools_flagstat.txt</code> was created using the <code>samtools flagstat</code> function.</p>
<div class="sourceCode" id="cb52"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb52-1" data-line-number="1"><span class="kw">library</span>(tidyverse)</a>
<a class="sourceLine" id="cb52-2" data-line-number="2"></a>
<a class="sourceLine" id="cb52-3" data-line-number="3">tcga &lt;-<span class="st"> </span><span class="kw">read.delim</span>(<span class="st">&quot;data/TCGA-LAML-multiqc_samtools_flagstat.txt&quot;</span>)</a>
<a class="sourceLine" id="cb52-4" data-line-number="4">info_tcga &lt;-<span class="st"> </span><span class="kw">subset</span>(tcga,<span class="dt">select=</span><span class="kw">c</span>(<span class="st">&quot;Sample&quot;</span>,<span class="st">&quot;mapped_passed&quot;</span>,<span class="st">&quot;total_passed&quot;</span>,<span class="st">&quot;total_failed&quot;</span>,<span class="st">&quot;secondary_passed&quot;</span>,<span class="st">&quot;secondary_failed&quot;</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb52-5" data-line-number="5"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">total_reads =</span> (total_failed <span class="op">-</span><span class="st"> </span>secondary_failed) <span class="op">+</span><span class="st"> </span>(total_passed <span class="op">-</span><span class="st"> </span>secondary_passed),</a>
<a class="sourceLine" id="cb52-6" data-line-number="6">         <span class="dt">prop_mapped_passed =</span> (mapped_passed <span class="op">-</span><span class="st"> </span>secondary_passed)<span class="op">/</span>total_reads)</a>
<a class="sourceLine" id="cb52-7" data-line-number="7"></a>
<a class="sourceLine" id="cb52-8" data-line-number="8">info_tcga<span class="op">$</span>total_passed_no_secondary &lt;-<span class="st"> </span>info_tcga<span class="op">$</span>mapped_passed <span class="op">-</span><span class="st"> </span>info_tcga<span class="op">$</span>secondary_passed</a>
<a class="sourceLine" id="cb52-9" data-line-number="9"><span class="kw">summary</span>(info_tcga<span class="op">$</span>total_passed_no_secondary<span class="op">/</span><span class="dv">2</span><span class="op">/</span><span class="dv">1000000</span>) <span class="co"># divide by two as flagstats reports the number of reads not fragments</span></a></code></pre></div>
<pre><code>##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##   36.70   54.98   58.68   57.72   61.53   69.62</code></pre>
<div class="sourceCode" id="cb54"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb54-1" data-line-number="1">tot_reads &lt;-<span class="st"> </span>info_tcga <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb54-2" data-line-number="2"><span class="st">  </span>dplyr<span class="op">::</span><span class="kw">select</span>(Sample,total_reads) <span class="op">%&gt;%</span><span class="st"> </span><span class="co"># total_reads is the sum of both Reads1 and Reads2</span></a>
<a class="sourceLine" id="cb54-3" data-line-number="3"><span class="st">  </span><span class="kw">mutate</span>(<span class="dt">Prop40 =</span> <span class="kw">round</span>((<span class="dv">80000000</span><span class="op">/</span>total_reads)<span class="op">*</span><span class="fl">1.24</span>,<span class="dv">2</span>)) <span class="op">%&gt;%</span></a>
<a class="sourceLine" id="cb54-4" data-line-number="4"><span class="st">  </span><span class="kw">separate</span>(Sample, <span class="dt">into =</span> <span class="kw">c</span>(<span class="st">&quot;Name&quot;</span>),<span class="dt">sep=</span><span class="st">&quot;[.]&quot;</span>) <span class="op">%&gt;%</span><span class="st">   </span><span class="co"># 80000000 = target number of reads, double the fragments </span></a>
<a class="sourceLine" id="cb54-5" data-line-number="5"><span class="st">  </span>dplyr<span class="op">::</span><span class="kw">select</span>(Name,Prop40,total_reads) </a>
<a class="sourceLine" id="cb54-6" data-line-number="6"></a>
<a class="sourceLine" id="cb54-7" data-line-number="7"><span class="kw">write_csv</span>(tot_reads,<span class="st">&quot;data/downsampling-proportion-TCGA-LAML.csv&quot;</span>)</a></code></pre></div>
<ul>
<li>Script to downsample at a certain proportion</li>
</ul>
<div class="sourceCode" id="cb55"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb55-1" data-line-number="1"><span class="ex">module</span> load picard-tools/2.9.4</a>
<a class="sourceLine" id="cb55-2" data-line-number="2"><span class="ex">module</span> load samtools/1.6</a>
<a class="sourceLine" id="cb55-3" data-line-number="3"></a>
<a class="sourceLine" id="cb55-4" data-line-number="4"><span class="ex">DownsampleSam</span> I=input.bam O=downsampled_output.bam PROBABILITY=sample_specific_prop RANDOM_SEED=200</a>
<a class="sourceLine" id="cb55-5" data-line-number="5"></a>
<a class="sourceLine" id="cb55-6" data-line-number="6"><span class="co"># Used to check the size of the doensampled file</span></a>
<a class="sourceLine" id="cb55-7" data-line-number="7"><span class="ex">samtools</span> flagstat downsampled_output.bam</a></code></pre></div>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="standardise-variants-output.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="ch-refer.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Supporting-libsize-scripts-book.pdf"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
